schema/common.gql

```diff
@@ -469,6 +469,44 @@ type ConfigMap implements Node & KubernetesResource {
   events: EventConnection! @goField(forceResolver: true)
 }
 
+"`ObjectReference` contains enough information to let you inspect or modify the referred object."
+type ObjectReference {
+  "Kind of the referent."
+  kind: String
+
+  "Namespace of the referent."
+  namespace: String
+
+  "Name of the referent."
+  name: String
+
+  "UID of the referent."
+  uid: String
+
+  "API version of the referent."
+  resourceVersion: String
+
+  """
+  If referring to a piece of an object instead of an entire object, this string
+  should contain a valid JSON/Go field access statement, such as desiredState.manifest.containers[2].
+  For example, if the object reference is to a container within a pod, this would take on a value like:
+  "spec.containers{name}" (where "name" refers to the name of the container that triggered
+  he event) or if no container name is specified "spec.containers[2]" (container with
+  index 2 in this pod). This syntax is chosen only to have some well-defined way of
+  referencing a part of an object.
+  """
+  fieldPath: String
```

Comment:
I think we should leave these out. Some context in https://github.com/crossplane/crossplane-runtime/issues/49.